bootstrap/pesticide_seed.py

```
@@ -0,0 +1,1179 @@
+    for row in rows:
+        if row.raw_row_hash == candidate.raw_row_hash:
+            return row
+    return None
+
+
+def upsert_product(
+    session: Session,
+    caches: UpsertCaches,
+    candidate: Product,
+    now: str,
+    append_mode: bool,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)